src/dotnet/ToolPackage/CommandSettingsListIdVersionRange.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.Frameworks;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.ToolPackage
+{
+    /// <summary>
+    ///     A range of CommandSettingsListId that is only different in the Version field.
+    /// </summary>
+    internal class CommandSettingsListIdVersionRange
+    {
+        public CommandSettingsListIdVersionRange(
+            PackageId packageId,
+            VersionRange versionRange,
+            NuGetFramework targetFramework,
```

Comment:
Add CommandName.

Reply:
Also rename class similar to previous one.

Reply:
Fixed, add in CommandName as key